mappings/net/minecraft/client/render/model/WrapperBakedModel.mapping

```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/class_10200 net/minecraft/client/render/model/WrapperBakedModel
```

Comment:
Is this a typo? Just mentioning since the field and constructor parameter are both named `wrapped` rather than `Wrapper`.

Reply:
Not a typo, the class is a wrapper for the wrapped model, but it could really be named either way /shrug